### PR TITLE
#6939 Improving exception messages.

### PR DESCRIPTION
**Pull request by burzum**

### CODE REVIEW

File: src/Database/Schema/Table.php [hunk 1]
```diff
@@ -403,18 +403,18 @@ public function addIndex($name, $attrs)
...
         if (empty($attrs['columns'])) {
-            throw new Exception('Indexes must have at least one column.');
+            throw new Exception(sprintf('Index `%s` in table `%s` must have at least one column.', $name,  $this->_table));
         }
         $attrs['columns'] = (array)$attrs['columns'];
         foreach ($attrs['columns'] as $field) {
             if (empty($this->_columns[$field])) {
                 $msg = sprintf(
-                    'Columns used in indexes must be added to the Table schema first. ' .
+                    'Columns used in index `%s` in table `%s` must be added to the Table schema first. ' .
```
Comment (ADmad) [inline src/Database/Schema/Table.php]: We normally use double quotes instead of back ticks as seen in line below.

**Comment by dereuromark** [comment]
Why not 3.0?

**Comment by burzum** [comment]
@dereuromark Cherry-pick to 3.0 if you want it in there? :)

**Comment by markstory** [comment]
Outside of the PHPCS errors this looks good.

**Pull request merged**